src/Key.js

```diff
@@ -2,14 +2,32 @@ import styled from "styled-components";
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 
-class Key extends Component {
-  setKeyColor = isInPractice => `
+const setKeyColor = isInPractice => `
     background: ${isInPractice ? "lightskyblue" : "lavenderblush"};
```

Comment:
I thought the convention was to put curly brackets and the return statement around the function if it was more than one line? What was the reason you wrote the function in this way?

Reply:
We use Prettier.js for our formatting, and it updates on save (in my IDE), so, for the most part, these style decisions are made by Prettier.

Prettier's formatting optimizes for dense code to minimize vertical spacing, and it pretty aggressively removes unnecessary formatting characters (like optional parentheses and braces).

I prefer to leave them in, personally.